GitUpKit/Utilities/NSColor+GINamedColors.m

```diff
@@ -116,6 +116,7 @@ +(NSColor*)gitUp##__NAME__##Color {
 IMPLEMENT_NAMED_COLOR(ConfigConflictBackground, "config/conflict_background", 1, 0.95, 0.95, 1)
 IMPLEMENT_NAMED_COLOR(ConfigGlobalBackground, "config/global_background", 0.95, 1, 0.95, 1)
 IMPLEMENT_NAMED_COLOR(ConfigHighlightBackground, "config/highlight_background", 1, 1, 0, 0.5)
+IMPLEMENT_NAMED_COLOR(CommitHeaderBackground, "commit/header_background", 0.5, 0.5, 0.5, 1)
```

Comment:
Was this not being set before? This asset did exist before this change no?

Reply:
It only lived in the nib (as an asset catalog reference) before, the only such use in the project. 10.13 appears to be decode it as black once a nib has been built with the newer Xcode. It appears it might only be looking in the main bundle rather than the framework bundle, but I was debugging blind in a VM so I don't know for sure.